server/routes/auth.ts

```diff
@@ -0,0 +1,26 @@
+import express, { Express, Request, Response, NextFunction, Router } from 'express';
+const router: Router = express.Router();
+import passport from 'passport';
+// const authController = require('../controllers/authController');
+// MLCK: somehow this line is needed but I'm not quite sure why!
```

Comment:
I think I remember one of the instructors saying in a lecture that the file is run when it is required in, so maybe that's why this line is needed.